packages/common/node-std/project.json

```diff
@@ -7,12 +7,38 @@
       "options": {
         "bundlePackages": [
           "assert",
+          "available-typed-arrays",
```

Comment:
wow, why are all these needed? are transitive deps not automatically bundled? I suppose that makes sense...